Sources/XCHammer/XcodeTarget.swift

```diff
@@ -1500,7 +1501,12 @@ public class XcodeTarget: Hashable, Equatable {
             First("${PYTHONPATH}:$(PROJECT_FILE_PATH)/XCHammerAssets")
         settings <>= getDeploymentTargetSettings()
 
-        let bazelScript = ProjectSpec.BuildScript(path: nil, script: getScriptContent(), name: "Bazel build")
+        let bazelScript = ProjectSpec.BuildScript(
+            path: nil,
+            outputFiles: [settings.infoPlistFile?.v ?? ""],
```

Comment:
Not sure if there's a better way to access the unwrapped value of this, open to suggestions